tests/workspace_bridge/test_bench_5007_arrow_fdd.py

```
@@ -0,0 +1,238 @@
+            "code": store.list_rules()[0]["code"],
+            "bindings": {"equipment_ids": ["bench-1"]},
+            "enabled": True,
+        },
+        saved_by="test",
+    )
+    model = MS().load()
+    rule = next(r for r in store.list_rules() if r.get("id") == "bench-eq-flatline")
+    bricks, points, eq = _plot_scope_for_columns(model, "demo", ["oa-t", "stat_zn-t", "oa-h"])
+    assert "bench-1" in eq
+    assert _rule_matches_plot_scope(rule, scope_bricks=bricks, scope_point_ids=points, scope_equipment_ids=eq)
+    _, points_duct, eq_duct = _plot_scope_for_columns(model, "demo", ["duct-t"])
```

Ruff (0.15.15)

[warning] Unpacked variable `points_duct` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)